install.sh

```diff
@@ -167,6 +168,7 @@ while getopts "hsinaf" opt; do
 		"f") overwrite_backup=true ;;
 		"?")
 			_bash-it_show_usage >&2
+			# shellcheck disable=SC2317
 			exit 1
```

Comment:
So the warning is not wrong, `_bash-it_show_usage` does exit on its own ...
But it exits with `0`.
If we see value with `exit 1` we should either modify `_bash-it_show_usage` to either NOT exit, or take the exit code as an optional argument.

Disabling the warning doesn't help because it will indeed never reach the `exit 1`

My recommendation is to remove the exit from the function.

These are the only two usages and both have their own exit, which seems appropriate to me so that we can vary the exit code based on context.

Reply:
I see it is already implemented in the function.